uktrade/data-hub-frontend
# Added a US States filter to Companies main page

## Description of change

New requirements mean that if a user selects USA as the location of their company, the US State must be filled out in the address section. 

This filter has been added to allow users to filter by US State.  

This feature is hidden behind a feature flag called `state-filter`. 

## Test instructions

- In Django Admin, add a feature flag called `state-filter`. Set it to active. 
- On DataHub, go to the Companies tab. 
- On the left side of the page, underneath the UK region filter, there should be a filter by the name of US state.
- The US State dropdown should be populated with 52 options. 
- Multiple State options can be selected, and when selected the companies displayed should now be filtered. 

## Screenshots
### Before

_Add a screenshot_

### After

_Add a screenshot_

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
